rotkehlchen/tests/utils/api.py

```diff
@@ -68,7 +68,7 @@ def assert_proper_response(
         response.headers["Content-Type"] == "application/json"
     )
     if status_code:
-        assert response.status_code == status_code
+        assert response.status_code == status_code, response.json()
```

Comment:
this looks weird. What are you doing here? If you want to include response data in the assertion during tests make it clear
```suggestion
        assert response.status_code == status_code, f'Response contains unexpected status code. Details {response.json()}'
```

Reply:
Yeah, looks better. I just added it to the pr because I was doing it locally too often during tests debugging